libs/ui/src/components/NeoAvatar/NeoAvatar.vue

```diff
@@ -1,28 +1,40 @@
 <template>
-  <NuxtImg
+  <TheImage
     v-if="avatar"
+    :image-component="imageComponent"
     :src="avatar"
     :alt="name"
     class="border neo-avatar"
     :width="size"
     :height="size"
+    :style="customStyle"
     @error="onError" />
   <img
     v-else
     :src="placeholder"
     :width="size"
     :height="size"
+    :style="customStyle"
     class="border neo-avatar" />
 </template>
 
 <script setup lang="ts">
+import type { ImageComponent } from '../TheImage/TheImage.vue'
+import TheImage from '../TheImage/TheImage.vue'
+
 const props = defineProps<{
+  imageComponent?: ImageComponent
   avatar?: string
   placeholder: string
   name: string
   size: number
```

Comment:
no need default value for size?

Reply:
It wasn't there before, it's a must-have property